### PR TITLE
2021.04+imx 5.10.35 2.0.0 fio: add imx6ulevk

### PR DESCRIPTION
**Pull request by MrCry0**
add imx6ul-evk support

### CODE REVIEW

File: arch/arm/mach-imx/mx6/Kconfig [hunk 1]
```diff
@@ -61,7 +61,7 @@ config MX6SLL
 
 config MX6UL
 	bool "i.MX 6UltraLite SoC support"
-	select HAS_CAAM
+	select HAS_CAAM if !OPTEE
```
Comment (ricardosalveti) [inline arch/arm/mach-imx/mx6/Kconfig]: In theory this shouldn't really cause any issue as long the jr device tree node is marked as reserved by the secure world (done by optee), so we should really investigate why here (good to merge as is, but please create a jira so we can track it later).

**Review by ricardosalveti — APPROVED** [review]
LGTM, minor comment but we can follow up later as the same change was already merged in 2020.04.

File: arch/arm/mach-imx/mx6/Kconfig (hunk 1 above)
Comment (MrCry0) [reply arch/arm/mach-imx/mx6/Kconfig]: Thanks, this fix shouldn't have been there. Fixed.

**Review by vanmaegima — APPROVED** [review]
LGTM

**Pull request merged**